app/features/quizzify/tools.py

```diff
@@ -18,9 +18,9 @@
 from langchain_google_genai import GoogleGenerativeAI
 from langchain_google_genai import GoogleGenerativeAIEmbeddings
 
-from app.services.logger import setup_logger
```

Comment:
You must not need to remove the app folder because it allows to avoid dependency issues.